includes/CLI.php

```diff
@@ -78,11 +56,6 @@ public static function orders( $args, $assoc_args ) {
 
 		$time_start = microtime( true );
 
-		$amount = (int) $amount;
```

Comment:
@coreymckrill I'm curious about why you removed parameter validation code?

Reply:
I should have included an example of an invalid paramter:

```
# wp wc generate orders '[1]'     
Generating orders  100% [===========================================================================================================================================] 0:14 / 0:00^C
```

Reply:
It was a brainfart, really. I realized that the `empty( $amount )` check wasn't needed because it gets set with the default value from the synopsis if it's empty. So I overzealously removed the whole thing. Thanks for catching that. Added back in 2c93f56

Reply:
(Also, two of the CLI commands already didn't have that validation, which probably made me think it wasn't necessary... without really thinking about it 🤦 )